modules/social_features/social_group/social_group.tokens.inc

```diff
@@ -75,13 +75,20 @@ function social_group_tokens($type, $tokens, array $data, array $options, Bubble
                 // If it's node.
                 case 'node':
                   $content_type = $entity->bundle();
-                  $article = 'a';
 
-                  if (preg_match('/^[aeioquy]/i', $content_type)) {
-                    $article .= 'n';
+                  // When a name of content name starts from a vowel letter then
+                  // will be added "an" before this name. For example "an
+                  // event".
+                  if (preg_match('/^[aeiou]/', $content_type)) {
```

Comment:
What about `an hour`? And `a helpful comment`?

Reply:
In order to handle this well we need to add proper support for indefinite articles. We can take a look at how this is done elsewhere, an example being https://raw.githubusercontent.com/Kaivosukeltaja/php-indefinite-article/master/IndefiniteArticle.class.php.

Maybe we should take this out of this PR and add a different one for solving that specific issue, what do you think?

Reply:
We should find an external library to do this for us and also take care of other languages (maybe there's a module for that?)

Reply:
I'll merge this in for now, as it is already a bit of an improvement. I will open a new PR for fixing the indefinite articles properly.